lib/crypto/test/crypto_SUITE.erl

```diff
@@ -1027,7 +1027,7 @@ group_config(rsa = Type, Config) ->
                   rsa_oaep(),
                   no_padding()
                  ],
-    Generate = [{rsa, 1024, 3}, {rsa, 2048, 17}, {rsa, 3072, 65537}],
+    Generate = [{rsa, 2048, 3}, {rsa, 3072, 65537}],
```

Comment:
With fips mode 2k gives error, but 3k does not.   The complaint is on the e value, not the modulus wich I didn't expect.  Does FIPS prescribe a different algorithm for generate_key so the e becomes invalid?
```
14> crypto:enable_fips_mode(false).                                                                                                              
true
15> f(),{Pub,Priv} = crypto:generate_key(rsa, {2048,3}), S = crypto:sign(rsa, sha256, <<"Hello world">>, Priv), crypto:verify(rsa, sha256, <<"Hello world">>, S, Pub).     
true
16> f(),{Pub,Priv} = crypto:generate_key(rsa, {3072,65537}), S = crypto:sign(rsa, sha256, <<"Hello world">>, Priv), crypto:verify(rsa, sha256, <<"Hello world">>, S, Pub).
true
17> 
17> 
17> crypto:enable_fips_mode(true).                                                                                                               true                     
18> f(),{Pub,Priv} = crypto:generate_key(rsa, {2048,3}), S = crypto:sign(rsa, sha256, <<"Hej hopp">>, Priv), {S, crypto:verify(rsa, sha256, <<"Hej hopp">>, S, Pub)}.     
** exception error: {error,{openssl,[{"rsa routines","RSA_BUILTIN_KEYGEN",
                                      "bad e value"}]}}
     in function  crypto:rsa_generate_key_nif/2
        called as crypto:rsa_generate_key_nif(2048,<<3>>)
     in call from crypto:generate_key/3 (crypto.erl, line 456)
19> f(),{Pub,Priv} = crypto:generate_key(rsa, {3072,65537}), S = crypto:sign(rsa, sha256, <<"Hello world">>, Priv), crypto:verify(rsa, sha256, <<"Hello world">>, S, Pub).
true
20> crypto:info_lib().
[{<<"OpenSSL">>,268439711,
  <<"OpenSSL 1.0.1i-fips 6 Aug 2014">>}]
21> 
```